website/content/en/docs/config/network/_index.md

```diff
@@ -69,10 +69,17 @@ The configuration steps are different for each network type:
 #### Managed (192.168.105.0/24)
 
 [`socket_vmnet`](https://github.com/lima-vm/socket_vmnet) is required for adding another guest IP that is accessible from the host and other guests.
+It must be installed according to the instruction provided on https://github.com/lima-vm/socket_vmnet.
+
+Note that installation using Homebrew is not secure and not recommended by the Lima project.
+Homebrew installation will only work with Lima if password-less `sudo` is enabled for the current user.
+The `limactl sudoers` command requires that `socket_vmnet` is installed into a secure path only
+writable by `root` and will reject `socket_vmnet` installed by Homebrew into a user-writable location.
 
 ```bash
-# Install socket_vmnet
-brew install socket_vmnet
+# Install socket_vmnet via MacPorts or from source
+# using instructions on https://github.com/lima-vm/socket_vmnet
```

Comment:
Readers still may refer to the Homebrew instructions there

Reply:
Probably better to list the entire commands (git clone, cd, git checkout vX.X.X, make, sudo make install.bin)

Reply:
Added all the commands, and also include the alternate instructions from https://github.com/lima-vm/socket_vmnet/issues/57#issue-2565168582 for installing from the tarball in case the user doesn't have XCode CLI tools installed.